fixes a bug introduced by changes upstream in the apt module.

bobweston opened the pull request:
This Pull Request addresses bug introduced by updated apt module.

It appears that ruby 1.9.3 needs to be removed from travis along with STRICT_VARIABLES is no longer supported in APT Module v2.1

That seems to be the cause of the failed tests

See #607 for more detail on what this addresses.

Commit:
fixes a bug introduced by changes upstream in the apt module.

Cosaquee commented:
#634 #595 #461 #528 all address the same issue. @garethr can you maybe close the duplicates after choosing one PR that fits you and tell us what should be done to merge fix like this?

bittner commented:
Same for #675. @garethr is there a chance to act on this problem?

Commit:
Added custom docker fact
  - Currently the implementation creates hash of the current docker networks
  - Also adds a docker.network.managed_interfaces hash to correspond a docker
    network with real interface

Commit:
updates the docker.rb file to match puppetlabs. 'Added custom docker fact
  - Currently the implementation creates hash of the current docker networks
  - Also adds a docker.network.managed_interfaces hash to correspond a docker
    network with real interface'
